lua/lvim/lsp/config.lua

```diff
@@ -4,16 +4,28 @@ return {
     signs = {
       active = true,
       values = {
-        { name = "LspDiagnosticsSignError", text = "" },
-        { name = "LspDiagnosticsSignWarning", text = "" },
-        { name = "LspDiagnosticsSignHint", text = "" },
-        { name = "LspDiagnosticsSignInformation", text = "" },
+        { name = "DiagnosticSignError", text = "" },
+        { name = "DiagnosticSignWarn", text = "" },
+        { name = "DiagnosticSignHint", text = "" },
+        { name = "DiagnosticSignInfo", text = "" },
       },
     },
     virtual_text = true,
     update_in_insert = false,
     underline = true,
     severity_sort = true,
+    float = {
+      source = "always",
```

Comment:
should we add
```lua
      border = "rounded",
      focusable = false,
```
here as well or you prefer the current one?

Reply:
~~do we have a border setting somewhere that we can re-use?~~
I'll switch it to use `lvim.lsp.popup_border`